carapace-server/src/main/java/org/carapaceproxy/api/CertificatesResource.java

```diff
@@ -220,6 +271,7 @@ public Response uploadCertificate(
             }
             CertificateData cert = new CertificateData(domain, "", encodedData, state, "", "", available);
             cert.setManual(MANUAL.equals(certType));
+            cert.setDaysAdvanceRenewal(daysadvancerenewal >= 0 ? daysadvancerenewal : DEFAULT_DAYS_ADVANCE_RENEWAL);
```

Comment:
If the user passes a negative value? I would expect an error, not an automatic adjustment